sklearn/mixture/dpgmm.py

```diff
@@ -566,18 +566,31 @@ def fit(self, X, y=None):
                                             np.trace(self.scale_[k]))
                 self.bound_prec_ *= 0.5
 
-        logprob = []
+        # EM algorithms
+        current_log_likelihood = None
         # reset self.converged_ to False
         self.converged_ = False
+
+        # this line should be removed when 'thresh' is deprecated
```

Comment:
can you maybe also add the version when it will be removed in the comment? I tend to grep for that ;)